### PR TITLE
update Crumble to version  0.82

### PR DESCRIPTION
**Pull request by maarten-k**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

Also added the doi and the license(is this the best way to describe a multi-license package? )

### CODE REVIEW

File: recipes/crumble/meta.yaml [hunk 1]
```diff
@@ -27,10 +27,15 @@ test:
...
-  #licence in describing paper is'OpenSource'. Asked for a specific type of license upstream https://github.com/jkbonfield/crumble/issues/7
-  license: open source
+  license: multiple BSD style licenses
+  license_family: BSD
+  #licence file is upstream in github, but not in release
+  #license_file: LICENSE.md
   summary: Controllable lossy compression of BAM/CRAM files
 
-extra: 
+identifiers:
```
Comment (dpryan79) [inline recipes/crumble/meta.yaml]: Please put this under `extra:`

Comment (maarten-k) [reply recipes/crumble/meta.yaml]: fixed. It might be a nice feature of linting to catch not wanted sections.

Comment (dpryan79) [reply recipes/crumble/meta.yaml]: Good idea

**Comment by epruesse** [comment]
@biocondabot recheck

File: recipes/crumble/meta.yaml (hunk 1 above)
Comment (epruesse) [reply recipes/crumble/meta.yaml]: @maarten-k Can you create an issue for that on bioconda-utils?

**Comment by epruesse** [comment]
LGTM

@biocondabot merge

**Pull request merged**